configs/components/oasis3mct/oasis3mct.env.yaml

```diff
@@ -14,7 +14,7 @@ add_compiletime_environment_changes:
                                 FCFLAGS: '"-fPIC"'
                 levante:
                         add_export_vars:
-                                OASIS_FFLAGS: '"-fPIC"'
+                                OASIS_FFLAGS: '"-march=core-avx2 -mtune=core-avx2 -fPIC"'
```

Comment:
Do you really need this change? As far as I know this was just resulting into a lot of warnings during compilation. @JanStreffing?

Reply:
You are right, these changes are not needed. I tried to undo them with a later commit